models/grading/fact_grade_status.sql

```diff
@@ -0,0 +1,64 @@
+with
+    latest_grade_status as (
+        select
+            org, course_key, object_id, actor_id, max(emission_time) as last_grade_time
+        from {{ ref("grading_events") }}
+        where
+            verb_id in (
+                'http://adlnet.gov/expapi/verbs/passed',
+                'http://adlnet.gov/expapi/verbs/failed'
+            )
+        group by org, course_key, object_id, actor_id
+    ),
+
+    latest_grades as (
+        select
+            org,
+            course_key,
+            object_id,
```

Comment:
Does `object_id` contain any information that `course_key` does not? I'm not able to generate these events locally so I don't know what this value looks like for passed/failed events.

Reply:
No, the object_id isn't necessary really